msprime/mutations.py

```diff
@@ -199,6 +199,285 @@ def __init__(self, state_independent=False):
         super().__init__(alleles, root_distribution, transition_matrix)
 
 
+def gamma_microsat(m, i, j, lo, hi):
```

Comment:
maybe these helper functions should be defined internally to the `general_microsat_rate_matrix` function?

Reply:
👍